adapter-guide/develop-stix-adapter.md

```diff
@@ -0,0 +1,602 @@
+# Developing a new STIX-shifter adapter
+
+* [Introduction](stix-adapter-overview.md)
+* [Scenario](#scenario)
+* [Prerequisites](#prerequisites)
+* [Steps](#steps)
+
+## Scenario
+{: #scenario}
+
+### Participants
+This scenario involves a software developer (*Developer A*) and an end user (*User A*). *Developer A* wants to implement a new adapter for the STIX-shifter project that can support a particular security product (*Product A*). *User A* is another developer that uses the STIX-shifter library.
+
+### Problem to solve
+*User A* performs security monitoring with *Product A* and several other security products. The other products already have existing STIX-shifter adapters. 
+
+*User A* would like to:
+1.	Submit one STIX pattern to query all the user’s security products at once. The use of a STIX pattern simplifies the search process because *User A* does not need to know the query language or API calls for each security product. 
+1.	See the query results from all the security products in one unified format (STIX bundle). With the assumption that the submitted pattern represents a potential security incident, the STIX bundle presents the query results in the context of the security event. 
+
+By implementing a new adapter, *Developer A* allows *Product A* to fit into the workflow.  
+
+
+## Prerequisites for *Developer A*
```

Comment:
As per the comment above, I think this just needs to be \#\# Prerequisites for the above link to work.